app/models/compound_aliquot.rb

```diff
@@ -20,9 +20,9 @@ class CompoundAliquot
   MULTIPLE_PROJECTS_ERROR_MSG =
     'Cannot create compound sample due to the component samples being under different projects.'
 
-  attr_accessor :request, :source_aliquots
+  attr_accessor :request, :source_aliquots, :compound_sample
```

Comment:
I saw that the above conversation was resolved, so sorry if I missed anything, but I think this `compound_sample` could go back to the `attr_reader` as it doesn't need to be set from outside any more?